test/powershell/Language/Scripting/HashtableToPSCustomObjectConversion.Tests.ps1

```diff
@@ -0,0 +1,248 @@
+﻿Describe "Tests for hashtable to PSCustomObject conversion" -Tags "CI" {
+
+    Context 'New-Object cmdlet should accept empty hashtable or $null as Property argument' {
+            
+        { $script:a = new-object psobject -property $null } | Should not throw
+        
+        It '$a should not be $null' { $script:a | Should Not Be $null }
+        It '$a type' { $script:a | should BeOfType "System.Management.automation.psobject" }
+    }

```

Comment:
fixed.

---
In reply to: [78253881](https://github.com/Powershell/Powershell/pull/2160#discussion_r78253881) [](ancestors = 78253881)